index.html

```diff
@@ -109,11 +109,10 @@ <h3 class="landing-subtitle">March 16-18, 2018</h3>
                 <h3 class="landing-subtitle">Indian Institute of Information Technology, Allahabad</h3>
             </div>
             <div class="row col-md-12 text-center wow fadeIn" data-wow-delay="1s">
-                <h4>Applications are open now</h4>
+                <h4>Applications are closed</h4>
             </div>
             <div class="row col-md-12 text-center wow fadeIn" data-wow-delay="1s">
                 <br>
-                <a class="btn btn-lg landing-btn" href="https://hackinthenorth.herokuapp.com" target="_blank">Apply</a>
```

Comment:
@Anupam-dagar but now people will miss link to herokuapp url. I would suggest to keep the button but change its text.